engineering_tools/xil-cloud-formation-v0.yaml

```diff
@@ -21,21 +21,25 @@ Parameters:
   ExistingVPCID:
     Description: The existing VPCId to use for the instance
     Type: String
-    Default: vpc-0b87aaf5689a49981
+    Default: vpc-00b15acce1e7ffb81
   ExistingRouteTableID:
     Type: String
-    Default: rtb-080bcd24bfb4b17a3
+    Default: rtb-03394845bff781cd9
   EC2SubnetCIDRBlock:
     Type: String
     Default: "10.0.0.0/24"
   CarmaConfigImage:
     Description: The specific carma config which launches xil and platform to make available in the instance
     Type: String
     Default: "usdotfhwastoldev/carma-config:xil-cloud-test"
-Mappings:
-  AMIMap:
-    us-east-1:
-      ami: ami-0c5b5f70bfa4f1021 # Custom Leidos AMI
+  CarmaConfigAMI:
+    Description: The AMI ID to use for the EC2 instance.
+    Type: String
+    Default:  ami-testing123 # Custom Leidos AMI ami-0c5b5f70bfa4f1021
```

Comment:
Is this also internal to us? Or can outside groups use this reference?

Reply:
This is a good catch. This ami is private and is based off a public ami. Unsure what changes were made on top of the base public image but we would probably need a minimum set of requirement for the rest of the resources in this deployment to work (Cloud9, NiceDCV, CDASIM?)